bin/kano-settings-onboot

```diff
@@ -1,353 +1,151 @@
-    Returns True if mode is changed
-    """
-    if status['group'] == edid['target_group'] and \
-       status['mode'] == edid['target_mode']:
-        logger.info('mode change not needed')
-        return False
-    else:
-        logger.info('mode change needed')
-        modes = '{} {}'.format(edid['target_group'], edid['target_mode'])
-        logger.info('setting mode: {}'.format(modes))
-
-        set_hdmi_mode(edid['target_group'], edid['target_mode'])
-        return True
-
-
-def compare_and_set_full_range(edid, status):
-    """
-    Returns True if full range is changed
-    """
-    if status['full_range'] == edid['target_full_range']:
-        logger.info('fullrange change not needed')
-        return False
-    else:
-        logger.info('fullrange change needed')
-        msg = 'setting fullrange to: {}'.format(edid['target_full_range'])
-        logger.info(msg)
-
-        if edid['target_full_range']:
-            set_config_value('hdmi_pixel_encoding', 2)
-        else:
-            set_config_value('hdmi_pixel_encoding', 0)
-        return True
-
-
-def compare_and_set_overscan(edid, status):
-    """
-    Returns True if overscan is changed
-    """
-    if status['overscan'] == edid['target_overscan']:
-        logger.info('overscan change not needed')
-        return False
-    else:
-        logger.info('overscan change needed')
-        logger.info('setting overscan to: {}'.format(edid['target_overscan']))
-
-        if edid['target_overscan']:
-            set_config_value('disable_overscan', 0)
-            overscan_value = -48
-        else:
-            set_config_value('disable_overscan', 1)
-            overscan_value = 0
-
-        set_config_value('overscan_left', overscan_value)
-        set_config_value('overscan_right', overscan_value)
-        set_config_value('overscan_top', overscan_value)
-        set_config_value('overscan_bottom', overscan_value)
-        return True
-
-
-def set_resolution_for_screen_kit(supported):
-    """
-    This function sets a predefined resolution for a Kano Screen Kit.
-
-    Returns:
-        changes - bool whether or not there were changes made
-    """
-    if is_screen_kit():
-        optimal_mode = find_best_resolution_aspect(supported, None, 1280, 800)
-        if optimal_mode:
-            if verbose:
-                print 'Applying SK optimal mode:', optimal_mode
-
-            logger.info('Applying SK optimal mode: {}'.format(optimal_mode))
-            set_hdmi_mode(optimal_mode['group'], optimal_mode['mode'])
-            return True
-        else:
-            logger.debug('Could not set the 1280x800 resolution for a SK!')
-
-    return False
-
-
-def set_resolution_for_unflashed_screen(status):
-    """
-    This function sets a predefined resolution for a display
-    without EDID information.
+from kano_settings.return_codes import RC_SUCCESS, RC_REQUIRES_ROOT, RC_INCORRECT_ARGS, \
+    RC_EDID_NO_MODEL, RC_EDID_NO_STATUS, RC_EDID_NO_SUPPORTED, RC_NO_EDID
```

Comment:
Isn't it more practical to just include them all?

Reply:
It might be, but the issue is that the IDE I'm using (Sublime Text) then cannot trace where some functions are getting imported from and cannot warn me if some functions have not been imported at all (because of `from x import *`).